frontend/src/main.ts

```diff
@@ -1,5 +1,5 @@
+import "reflect-metadata";
```

Comment:
Could you please write a comment describing why this has to be imported in `main.ts` if it's not used in this file?